new_lineage_panel/src/App.tsx

```diff
@@ -73,9 +78,15 @@ export const endProgressBar = () => {
 export const previewFeature = () => {
   vscode.postMessage({ command: "previewFeature", args: {} });
 };
+export const showNoLineage = (params: { table: string; name: string }) => {
+  vscode.postMessage({ command: "showNoLineage", args: { params } });
+};
 
 export let isDarkMode = false;
 
+let sessionId = uuidv4();
```

Comment:
Would be better to keep in separate file if it needs to be static. Otherwise, whenever this variable or `getSessionId` is referred, App.tsx will create circular dependency